specs/core/0_beacon-chain.md

```diff
@@ -1939,7 +1931,7 @@ def get_justification_and_finalization_deltas(state: BeaconState) -> Tuple[List[
     for index in eligible_validators:
```

Comment:
Should `eligible_validators` be `get_active_validator_indices(state.validator_registry, current_epoch)`?